### PR TITLE
Fix serializer validation

### PR DESCRIPTION
**Pull request by JobDoesburg**
### Summary
Thoroughly fixes model validation after API updates, and changes some things to the EventRegistrationAdminSerializer to make it more convenient.

### How to test
1. Try to POST, PUT and PATCH to the API, bot things that should and should not clean. For example, an Event Registration with both a `member` and `name` field.

### CODE REVIEW

File: website/events/api/v2/admin/serializers/event_registration.py [hunk 1]
```diff
@@ -25,14 +25,36 @@ class Meta:
             "name",
             "special_price",
             "remarks",
+            "queue_position",
```
Comment (se-bastiaan) [inline website/events/api/v2/admin/serializers/event_registration.py:28]: This is repeated

File: website/events/api/v2/admin/serializers/event_registration.py [hunk 2]
```diff
@@ -25,14 +25,36 @@ class Meta:
...
+    def _is_late_cancellation(self, instance):
+        return instance.is_late_cancellation()
+
+    def _queue_position(self, instance):
+        pos = instance.queue_position
+        return pos if pos and pos > 0 else None
 
-    payment = PaymentSerializer()
+    def _is_cancelled(self, instance):
+        return instance.date_cancelled is not None
```
Comment (se-bastiaan) [inline website/events/api/v2/admin/serializers/event_registration.py:53]: With `date_cancelled` available this is kind of redundant.

**Review by se-bastiaan — COMMENTED** [review]
Don't really have time to check if it actually works, but the cleaned model serializer does look good.

**Review by pingiun — APPROVED** [review]
Not sure if this should work but I want to test it out

**Pull request merged**